Book-A-Desk.Api/BookingsHttpHandler.fs

```diff
@@ -14,15 +15,26 @@ open Book_A_Desk.Domain.Office.Domain
 open Book_A_Desk.Domain.Reservation
 open Book_A_Desk.Domain.Reservation.Commands
 open Book_A_Desk.Domain.Reservation.Domain
+
 open Book_A_Desk.Infrastructure.DynamoDbEventStore
 
 type BookingsHttpHandler =
     {
         HandlePostWith: Models.Booking -> HttpHandler
     }
 
+type EmailDetails =
+    {
+        toAddress : string
+        fromAddress : string
+        body : string
+    }
+
+type MessageDetails =
```

Comment:
Is this still used? If not, please remove both types.

Reply:
Cleaned!